example.init.yaml

```diff
@@ -175,15 +175,16 @@ secrets:
     namespace: "openfaas-fn"
 
 ## If not using a HTTPS URL, then set a list of CUSTOMERS
+## To use this set "value_from" to a real file path, and put in a list (each item on a new line) of usernames, without other whitespace
   - name: "of-customers"
-    literals:
```

Comment:
Was this a typo on my part?

Reply:
I think so, if the intention was to read it from a file rather than a literal?